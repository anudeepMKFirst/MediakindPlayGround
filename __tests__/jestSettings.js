import React from 'react';
import SettingsModal from "./../settings.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<SettingsModal/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});