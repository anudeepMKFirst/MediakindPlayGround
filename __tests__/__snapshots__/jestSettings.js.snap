// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#202328",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
  testID="Settings Screen"
>
  <Image
    source={
      Object {
        "uri": "res://drawable/default/bokeh_bg_large.png",
      }
    }
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#202328",
        "bottom": 0,
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 200,
        "justifyContent": "flex-start",
        "left": 100,
        "margin": 40,
        "position": "absolute",
        "top": 200,
        "width": 400,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 72,
          "margin": 20,
          "opacity": 1,
          "padding": 20,
          "width": 400,
        }
      }
      testID="About Button"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 32,
          }
        }
      >
        ABOUT
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 72,
          "margin": 20,
          "opacity": 1,
          "padding": 20,
          "width": 400,
        }
      }
      testID="TOS Button"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 32,
          }
        }
      >
        TERMS OF SERVICE
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 72,
          "margin": 20,
          "opacity": 1,
          "padding": 20,
          "width": 400,
        }
      }
      testID="PP Button"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 32,
          }
        }
      >
        PRIVACY POLICY
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "bottom": 200,
        "justifyContent": "flex-start",
        "left": 600,
        "position": "absolute",
        "right": 200,
        "top": 200,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 48,
          "margin": 40,
          "textAlign": "left",
        }
      }
    >
      ABOUT
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "gray",
          "fontSize": 24,
          "height": 450,
          "marginLeft": 40,
          "marginRight": 40,
          "textAlign": "left",
        }
      }
    >
      RNSampleApp (Beautiful CLOUD over the react native world) - ver 1.0
    </Text>
  </View>
</View>
`;
